apps/prairielearn/src/migrations/20230807222115_stripe_checkout_sessions__create.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE IF NOT EXISTS
+  stripe_checkout_sessions (
+    id BIGSERIAL PRIMARY KEY,
+    -- TODO: better name?
+    session_id TEXT NOT NULL UNIQUE,
```

Comment:
Ideally I'd like to better disambiguate our table ID from the ID that Stripe itself uses to identify the session. Maybe `stripe_checkout_session_id` for the latter?

Reply:
I might actually go with `stripe_id`, my fear with `stripe_checkout_session_id` is that per our conventions, that would actually refer to this table itself! `stripe_id` *could* imply the existence of a `stripes` table, but I think "ID in Stripe" is the more natural interpretation. I'll make this change.

Reply:
Or maybe `stripe_object_id`?

Reply:
I went with `stripe_object_id`.